CefSharp.Core/Internals/CefBrowserHostWrapper.cpp

```diff
@@ -20,6 +21,40 @@ void CefBrowserHostWrapper::Print()
     _browserHost->Print();
 }
 
+void CefBrowserHostWrapper::PrintToPDF(String^ path, CefSharpPdfPrintSettings^ settings, IPrintToPdfCallback^ callback)
+{
+    CefPdfPrintSettings nativeSettings;
+    CefString(&nativeSettings.header_footer_title).FromString(StringUtils::ToNative(settings->HeaderFooterTitle));
```

Comment:
This can likely be simplified e.g.
https://github.com/cefsharp/CefSharp/blob/master/CefSharp.Core/BrowserSettings.h#L42